src/ray/object_manager/object_manager.cc

```diff
@@ -1,5 +1,12 @@
 #include "ray/object_manager/object_manager.h"
 
+// forward declaration.
+const std::vector<ray::ObjectID> from_flatbuf(
+    const flatbuffers::Vector<flatbuffers::Offset<flatbuffers::String>> &vector);
+flatbuffers::Offset<flatbuffers::Vector<flatbuffers::Offset<flatbuffers::String>>>
+to_flatbuf(flatbuffers::FlatBufferBuilder &fbb,
+           const std::vector<ray::ObjectID> &object_ids);
```

Comment:
Instead of forward declaring these, should we just include the relevant header files?